src/main/java/seedu/planitarium/ExpenditureList.java

```diff
@@ -0,0 +1,51 @@
+package seedu.planitarium;
+
+import java.util.ArrayList;
+
+public class ExpenditureList {
+
+    protected ArrayList<Expenditure> expenditureArrayList;
+
+    public ExpenditureList() {
+       this.expenditureArrayList = new ArrayList<>();
+    }
+
+    public void addExpenditure(String description, double amount) {
+        this.expenditureArrayList.add(new Expenditure(description, amount));
+    }
+
+    public double getExpenditureValue(int index) {
+        double value = 0;
+        try {
+            value =  expenditureArrayList.get(index - 1).amount;
+        } catch (IndexOutOfBoundsException e) {
```

Comment:
Buggy. You might want to use a parser method instead? Perhaps can throw some exception if it fails
As of now the code will still return 0 like nothing happened even though there is something printed
More similar situations below